### PR TITLE
Add IntelliJ metadata for node projects

### PR DESCRIPTION
**Pull request by etanshaul**
See #74 and #72 

This is the same idea, but for the NodeJS projects

### CODE REVIEW

**Comment by etanshaul** [comment]
@simonz130 ptal

File: nodejs/nodejs-hello-world/.idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/nodejs-hello-world.iml" filepath="$PROJECT_DIR$/.idea/nodejs-hello-world.iml" />
```
Comment (simonz130) [inline nodejs/nodejs-hello-world/.idea/modules.xml]: Are both parameters needed?

Comment (etanshaul) [reply nodejs/nodejs-hello-world/.idea/modules.xml]: You mean “fileurl” and “filepath”? I’m not totally sure, but these files were auto generated by the IDE so I lean towards leaving them as is.

Comment (simonz130) [reply nodejs/nodejs-hello-world/.idea/modules.xml]: Yeah, if it's auto-generated, let's leave them. I'm merging your PR.

Thanks!

**Pull request merged**